modules/data-entry/src/main/frontend/src/questionnaire/FileQuestion.jsx

```diff
@@ -282,7 +288,12 @@ function FileQuestion(props) {
 
   let hrefs = Array.of(existingAnswer?.[1]["value"]).flat();
   let defaultDisplayFormatter = function(label, idx) {
-    return <a href={fixFileURL(hrefs[idx], label)} target="_blank" rel="noopener" download>{label}</a>;
+    return (
+      <div>
+        <Link href={fixFileURL(hrefs[idx], label)} target="_blank" rel="noopener" download>{label}</Link>
```

Comment:
This is needed in now in mui5 to keep links looking consistent:
```suggestion
        <Link href={fixFileURL(hrefs[idx], label)} target="_blank" rel="noopener" download underline="hover">{label}</Link>
```